### PR TITLE
Adding a page about creating a professional portfolio

### PR DESCRIPTION
**Pull request by djwfyi**
Portfolios come up pretty frequently in the WTD Slack community in various channels.
This PR page adds some of the common discussion points that come up for creating portfolios.

The page is located in the Hiring Guide  folder and included in that page's Usefiul Skills heading.

----
:books: Documentation preview :books:: https://writethedocs-www--1904.org.readthedocs.build/

### CODE REVIEW

**Comment by silopolis** [comment]
How timely! 😍👌
Thanks a ton for creating this important piece 🙏🙏🙏

**Comment by djwfyi** [comment]
Comment from slack

- If you wish, you can use my website: https://www.aliamm.com/
- I'd suggest making the links go to new tabs or windows.
- As you indicate, building a website is a skill. You might suggest adding any skills from the process to the resume: HTML, CSS, Javascript, web-site builders, whatever...
- My 'portfolio' is a static site generated manually. You don't list that as a possibility.
- I'd put quotes around 'how to'. It's confusing to read otherwise.
- I'd also mention that entire documents are unlikely to be read. It's probably better to be selective about what to provide.
- Include the need for disclaimers about any web=based content.
- Also, many technical writers are not the sole creators of anything. So that may need to be clarified.

File: docs/hiring-guide/portfolios.rst [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+Most job searches require that technical writers produce one or more samples of their writing.
+Managers and other interviewers rely on such samples to determine the abilities of the applicant.
+Because nearly every job posting expects a writer to provide a writing sample, a tech writer on the job hunt should have one or more samples ready.
+
+What to include in a portfolio
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A portfolio consists of a set of the writer's best samples of their work.
+What "best samples" means is entirely subjective on your part.
+When building out a portfolio, include samples of writing that
```
Comment (HCloward) [inline docs/hiring-guide/portfolios.rst]:
```suggestion
When building out a portfolio, include samples of writing that:
```

File: docs/hiring-guide/portfolios.rst [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+- you are exceptionally proud of
+- demonstrate a particular skill
+- validates domain knowledge in a particular area
```
Comment (HCloward) [inline docs/hiring-guide/portfolios.rst]:
```suggestion
- validate domain knowledge in a particular area
```

File: docs/hiring-guide/portfolios.rst [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+- showcases your understanding of good technical writing technique
```
Comment (HCloward) [inline docs/hiring-guide/portfolios.rst]:
```suggestion
- showcase your understanding of good technical writing technique
```

File: docs/hiring-guide/portfolios.rst (hunk 2 above)
Comment (HCloward) [reply docs/hiring-guide/portfolios.rst]: I believe the noun these verbs are referring to is "samples" which is plural so you need the plural version of the verbs.

File: docs/hiring-guide/portfolios.rst [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+It's your portfolio.
+You get to decide.
+
+Portfolio examples
+~~~~~~~~~~~~~~~~~~
+
+A list of portfolio examples from Write the Docs folks who have agreed to share theirs here:
+
+- `Daryl White <https://www.djw.fyi>`__
+- `Alia Michaels <https://www.aliamm.com>`
```
Comment (HCloward) [inline docs/hiring-guide/portfolios.rst]:
```suggestion
- `Alia Michaels <https://www.aliamm.com>`
- `Heather Cloward <hcloward.github.io>`
```

**Review by HCloward — APPROVED** [review]
This is great and much needed! Thanks for taking the time to write it Daryl

File: docs/hiring-guide/portfolios.rst (hunk 4 above)
Comment (plaindocs) [reply docs/hiring-guide/portfolios.rst]:
```suggestion
- `Alia Michaels <https://www.aliamm.com>`__
- `Heather Cloward <hcloward.github.io>`__
```

even

File: docs/hiring-guide/portfolios.rst (hunk 1 above)
Comment (djwfyi) [reply docs/hiring-guide/portfolios.rst]: :+1:

File: docs/hiring-guide/portfolios.rst (hunk 2 above)
Comment (djwfyi) [reply docs/hiring-guide/portfolios.rst]: nice catch, thanks!

File: docs/hiring-guide/portfolios.rst (hunk 3 above)
Comment (djwfyi) [reply docs/hiring-guide/portfolios.rst]: :+1:

File: docs/hiring-guide/portfolios.rst (hunk 4 above)
Comment (djwfyi) [reply docs/hiring-guide/portfolios.rst]: Added!

**Review by plaindocs — APPROVED** [review]
I didn't edit this closely, but it looks great! 

Worth cross linking in https://writethedocs-www--1904.org.readthedocs.build/topics/#getting-hired  maybe?

**Comment by djwfyi** [comment]
> Worth cross linking

I think it's a good idea, but my skill at creating internal links in these docs is lacking. It didn't go well when I tried it for the SEO page. Eric wound up creating a top link (rather than a deep link) to the page I was referencing. If you know how to add a deep link to that heading, please do! Or tell me the syntax to use.
(I tried adding a `:ref:` tag and referencing that, but it didn't work on the SEO page.)

**Comment by plaindocs** [comment]
This works for me

```diff
diff --git i/docs/topics.rst w/docs/topics.rst
index 0313410a..929b29b5 100644
--- i/docs/topics.rst
+++ w/docs/topics.rst
@@ -291,6 +291,8 @@ Getting hired
 - |:newspaper:| `Creating a doc portfolio </blog/newsletter-september-2017/#doc-portfolios-a-perpetual-conundrum>`__
 - |:newspaper:| `Writing a resume </blog/newsletter-august-2017/#what-resume-advice-is-the-right-resume-advice>`__
 - |:newspaper:| `Putting together a portfolio </blog/newsletter-october-2016/#putting-together-a-technical-writing-portfolio>`__
+- |:newspaper:| `Maintaining a portfolio </hiring-guide/portfolios/>`__
```

and yes it's fiddly! Should maybe add a note in #1152

**Comment by plaindocs** [comment]
Or did you mean vice versa? I think this way is more useful.

**Comment by djwfyi** [comment]
OH! You meant adding a link to this new page to the topic guide. :smile:

**Comment by plaindocs** [comment]
I did indeed ;-)

File: docs/hiring-guide/portfolios.rst [hunk 5]
```diff
@@ -140,4 +140,4 @@ A list of portfolio examples from Write the Docs folks who have agreed to share
 Other resources
 ~~~~~~~~~~~~~~~
 
-The :doc:`topic index </topics/#getting-hired>` has links to several previous Write the Docs newsletter articles about portfolios.
```
Comment (plaindocs) [inline docs/hiring-guide/portfolios.rst]: <3

Comment (plaindocs) [reply docs/hiring-guide/portfolios.rst]: https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#cross-referencing-arbitrary-locations

Comment (djwfyi) [reply docs/hiring-guide/portfolios.rst]: Fixed.

**Pull request merged**